interface/forms/clinical_instructions/new.php

```diff
@@ -36,28 +37,61 @@
 ?>
 <html>
     <head>
-        <link rel="stylesheet" href="<?php echo $css_header; ?>" type="text/css">
+        <?php Header::setupHeader('bootstrap'); ?>
+        
+        <style type="text/css" title="mystyles" media="all">
+ 
+            .form-group{
+                margin-bottom: 5px;
+                
+            }
+            legend{
+                border-bottom: 2px solid #E5E5E5;
+                background:#E5E5E5;
+                padding-left:10px;
+            }
+            .form-horizontal .control-label {
+                padding-top: 2px;
+            }
+            fieldset{
+                background-color: #F2F2F2;
+                margin-bottom:10px;
+                padding-bottom:15px;
+            }
+        </style>
     </head>
     <body class="body_top">
-        <p><span class="forms-title"><?php echo xlt('Clinical Instructions Form'); ?></span></p>
-        </br>
-        <?php echo "<form method='post' name='my_form' " . "action='$rootdir/forms/clinical_instructions/save.php?id=" . attr($formid) . "'>\n"; ?>
-        <table style="">
-            <tr id="">
-                <td>
-                    <label>
-                        <?php echo xlt('Instructions').':'; ?>
-                    </label>
-                </td>
-                <td>
-                    <textarea name="instruction" id ="instruction" style="width:500px;height:100px;"><?php echo text($check_res['instruction']); ?></textarea>
-                </td>
-                <td>
-                    <input type='submit'  value='<?php echo xla('Save'); ?>' class="button-css">&nbsp; 
-                </td>
-            </tr>
-        </table>
-    </form>    
+        <div class="container">
+            <div class="row">
+                
+                    <div class="page-header">
+                        <h2><?php echo xlt('Clinical Instructions Form'); ?></h2>
+                    </div>
+                
+            </div>
+            <div class="row">
+                    <?php echo "<form method='post' name='my_form' " . "action='$rootdir/forms/clinical_instructions/save.php?id=" . attr($formid) . "'>\n"; ?>
+                        <fieldset>
+                        <legend class=""><?php echo xlt('Instructions'); ?></legend>
+                            <div class="form-group">
+                                <div class="col-sm-10 col-sm-offset-1">
+                                    <textarea name="instruction" id ="instruction"  class="form-control" cols="80" rows="5" ><?php echo text($check_res['instruction']); ?></textarea>
+                                </div>
+                            </div>
+                        </fieldset>
+                        <div class="form-group">
+                            <div class="col-sm-12 text-center">
```

Comment:
The reason for centering is because I was using <div class="container"> which centers the whole page and visually the centered buttons looked better balanced than having it pulled to the right